### PR TITLE
Resolved: 1196-humanitarian-issue-fix

### PR DESCRIPTION
**Pull request by PG-Momik**
This PR fixes issue #1196  task 4 and 5

### CODE REVIEW

File: app/Console/Commands/FixDocumentLink.php [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+        'language'      => [['language' => null]],
+        'document_date' => [['date'     => null]],
+    ];
+
+    /**
+     * @var
+     */
+    public $brokenCount = 0;
+
+    public function __construct(
```
Comment (sriza) [inline app/Console/Commands/FixDocumentLink.php:73]: add php docs

File: app/Console/Commands/FixDocumentLink.php [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+     * [aidstreamOrgId => iatiOrgId].
+     *
+     * @param array $aidstreamOrganizationIdentifierArray
+     * @param $iatiOrganizationIdArray
+     *
+     * @return array
+     */
+    private function mapOrganizationIds(array $aidstreamOrganizationIdentifierArray, $iatiOrganizationIdArray): array
+    {
+        $returnArr = [];
```
Comment (sriza) [inline app/Console/Commands/FixDocumentLink.php:209]: rename variable

File: app/Console/Commands/FixDocumentLink.php [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+    /**
+     * Returns array of [organizationId => organizationIdentifier].
+     *
+     * @param $aidstreamOrganizationIds
+     *
+     * @return array
+     */
+    private function getAidstreamOrganizationIdentifier($aidstreamOrganizationIds): array
+    {
+        $returnArr = [];
```
Comment (sriza) [inline app/Console/Commands/FixDocumentLink.php:180]: rename variable

**Review by sriza — CHANGES_REQUESTED** [review]

File: app/Console/Commands/FixDocumentLink.php [hunk 4]
```diff
@@ -0,0 +1,298 @@
...
+use App\IATI\Traits\MigrateActivityTrait;
+use App\IATI\Traits\MigrateOrganizationTrait;
+use Exception;
+use Illuminate\Console\Command;
+use Illuminate\Database\DatabaseManager;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Facades\DB;
+use Illuminate\Support\Facades\Validator;
+
+class FixDocumentLink extends Command
```
Comment (Sanilblank) [inline app/Console/Commands/FixDocumentLink.php:18]: add doc comment

File: app/Console/Commands/FixDocumentLink.php [hunk 5]
```diff
@@ -0,0 +1,298 @@
...
+        'language'      => [['language' => null]],
+        'document_date' => [['date'     => null]],
+    ];
+
+    /**
+     * @var int
+     */
+    public int $brokenCount = 0;
+
+    public function __construct(
```
Comment (Sanilblank) [inline app/Console/Commands/FixDocumentLink.php:73]: doc comment

**Review by sriza — APPROVED** [review]

**Pull request merged**